sample/src/main/java/com/klaviyo/sample/SampleApplication.kt

```
@@ -0,0 +1,31 @@
+package com.klaviyo.sample
+
+import android.app.Application
+import com.google.firebase.messaging.FirebaseMessaging
+import com.klaviyo.analytics.Klaviyo
+import com.klaviyo.analytics.model.EventMetric
+import com.klaviyo.forms.registerForInAppForms
+
+class SampleApplication : Application() {
+    override fun onCreate() {
+        super.onCreate()
+
+        // Initialize Klaviyo SDK: Add your public API key here or in the local.properties file
+        val klaviyoPublicKey = BuildConfig.KLAVIYO_PUBLIC_KEY
```

No shot this could be nullable, right? If so we could just throw an error here so developers pretty immediately know they missed that public key addition, not sure if that would be easier than sifting through the logs to find the klaviyo network error 🤷

---

Yeah good thinking. Even if not nullable, could also check string length or something